virttest/qemu_vm.py

```diff
@@ -1192,12 +1192,15 @@ def add_boot(devices, boot_order, boot_once, boot_menu, boot_strict):
                 logging.warn("-boot on s390x only support boot strict=on")
                 return "-boot strict=on"
             cmd = " -boot"
-            patterns = ["order", "once", "menu", "strict"]
+            patterns = ["order", "once", "menu", "strict", "reboot-timeout"]
```

Comment:
What about set "boot_reboot-timeout" as a param of 'add_boot', like 'def add_boot(devices, boot_order, boot_once, boot_menu, boot_strict, boot_reboot-timeout)'

Reply:
@sitoliu   qemu-kvm must support all params in add_boot.  As order,once,menu,strict,reboot-timeout are standard qemu-kvm params.  e.g  -boot menu=on,once=c......   If I add prefix boot_xx as param. qemu-kvm doesn't support.

Reply:
@xiangchunfu Thanks.